ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -223,12 +223,12 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "db_query_duration_over_t
   query = <<-QUERY
 dependencies
 ${local.skip_on_weekends}
-| where timestamp >= ago(5m) and name has "SQL:" and duration >= 1000
+| where timestamp >= ago(5m) and name has "SQL:" and duration > 1250
   QUERY
 
   trigger {
     operator  = "GreaterThan"
-    threshold = 4
+    threshold = 10
```

Comment:
Technically this should be `9` no? Otherwise we're checking `>= 11`

Reply:
I was trying to make the alert simpler by having the value in the title equal the value in the trigger. Happy to change the tittle to `> 10` instead of `10+` if that better illustrates the function of the alert

Reply:
Yeah it's not a huge deal either way, just noting that they currently do not match.